### PR TITLE
add status badge images to guide

### LINKED ISSUE #4137: Page on status badges does not contain any rendered examples
# Missing Information

Images of the badges as they actually render.

# Already Looked

Nowhere, really.

# Potential Locations

https://circleci.com/docs/2.0/status-badges/

### PR DESCRIPTION
**Pull request by rosieyohannan**
fixes #4137

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# CircleCI Documentation [![CircleCI Build Status](https://circleci.com/gh/circleci/circleci-docs.svg?style=shield)](https://circleci.com/gh/circleci/circleci-docs) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/circleci/circleci-docs/master/LICENSE) [![CircleCi Community](https://img.shields.io/badge/community-CircleCI%20Discuss-343434.svg)](https://discuss.circleci.com)
+# CircleCI Documentation 
+[![CircleCI Build Status](https://circleci.com/gh/circleci/circleci-docs.svg?style=shield)](https://circleci.com/gh/circleci/circleci-docs) 
+[![CircleCI Build Status](https://circleci.com/gh/circleci/tree/rosie/add-status-badge-images.svg?style=shield)](https://circleci.com/gh/circleci/circleci-docs)
```
Comment (rosieyohannan) [inline README.md]:
```suggestion
[![CircleCI Build Status](https://circleci.com/gh/circleci/tree/rosie/add-optimization-recipes-v2.svg?style=shield)](https://circleci.com/gh/circleci/circleci-docs) 
```

**Comment by rosieyohannan** [comment]
@teesloane I'm trying to (temporarily) add a badge to the readme for a failing branch so I can see the various status badges available. I can't get the branch specific badge working using your example here: https://circleci.com/docs/2.0/status-badges/#overview

Just wondering if you could have a quick look and see if I've made a silly mistake maybe? typo etc...

**Comment by teesloane** [comment]
@rosieyohannan The reason this is failing:

- the url grabs the badge based on the slug of the git branch. Because this branch has a `/` in it, (which creates subfolders in git) the url cannot be like so:

`https://circleci.com/gh/circleci/circleci-docs/tree/rosie/add-optimization-recipes-v2.svg?style=svg`
because `rosie/` creates a new url. So, branches with `/` in them are made url friendly by replacing the slash with a `-`, from what I can see. 

The following url works:

`https://circleci.com/gh/circleci/circleci-docs/tree/rosie-add-optimization-recipes-v2.svg?style=svg`

Making this badge: 

However, I've noticed that the branch slug for `add-optimizations-optimization-recipes` also works without the folder prefix. So, perhaps internally we are just using the the branch name after the folders names have been stripped.

**Comment by rosieyohannan** [comment]
@teesloane ah 'no builds' weeeeeird. I didn't even know there was a badge for that! And there definitely are builds. I will investigate...

**Comment by rosieyohannan** [comment]
This is strange, so I used the URL scape thing for `/` - `%2F` and now the badge says PASSED, when the branch is failing. Also the different badge types render with "PASSED" and "Passing" which feels weird. Passing sounds like it's still in progress (but that's a separate issue)

Screenshot 2020-02-14 at 14 32 07

**Comment by KyleTryon** [comment]
> This is strange, so I used the URL scape thing for `/` - `%2F` and now the badge says PASSED, when the branch is failing. Also the different badge types render with "PASSED" and "Passing" which feels weird. Passing sounds like it's still in progress (but that's a separate issue)
> 
> Screenshot 2020-02-14 at 14 32 07


Using the failing example branch you have here using the `%2F` replacement appears to work for me:


The branch name can be copied from the URL here and placed in the shield, as it will already have the slash converted.

https://app.circleci.com/github/circleci/circleci-docs/pipelines?branch=rosie%2Fadd-optimization-recipes-v2

The passing branch:



https://app.circleci.com/github/circleci/circleci-docs/pipelines?branch=rosie%2Fadd-status-badge-images

Looks like you had it 👍  let me know if I missed something.

**Pull request merged**